pytorch_lightning/loggers/base.py

```diff
@@ -25,22 +27,64 @@ def wrapped_fn(self, *args, **kwargs):
 class LightningLoggerBase(ABC):
     """Base class for experiment loggers."""
 
-    def __init__(self):
+    def __init__(self, metrics_agg_fn: Callable[[Sequence[Dict[str, float]]], float] = metrics_agg.metrics_agg_avg):
```

Comment:
this means that a single logger can do only one type of operation (max/avg/..) overall metrics?

Reply:
I agree, we should have at least the option to give a separate aggregation function per key (as a dict) and when only one function is given, we can still build an internal dict so that we always do a lookup there.